### PR TITLE
test: add sanity test cases to stability integration test

### PR DESCRIPTION
**Pull request by cslong**
*Issue #, if available:*

*Description of changes:*
- Add sanity checks to verify numbers and results from assessment and porting are what we expect.
- Changed test to wait on spinner class for so test won't accidentally move on too soon and get stuck
- Only port up to two projects to save time

*Testing done:*
- Manual running of test in workspace
- Created new code build, "integration-test" with the beta AWS account

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/porting-assistant-dotnet-ui/blob/develop/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including READMEs and comments (where appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific environment

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/integration-test/src/stability.spec.ts [hunk 1]
```diff
@@ -198,8 +312,11 @@ describe("stability check, assess a solution, reassess the solution, check all s
       "Miniblog.Core.sln"
     );
     await addSolution(app, solutionPath);
-    await app.stop();
-    app = await startApp();
-    return await runThroughSolution(solutionPath, "inplace");
+    await app.client.refresh();
+    const results = await runThroughSolution(solutionPath, "inplace");
```
Comment (huawenm) [inline packages/integration-test/src/stability.spec.ts:316]: as we discussed, let's parameterize the target framework

File: packages/integration-test/src/stability.spec.ts [hunk 2]
```diff
@@ -48,7 +53,8 @@ describe("stability check, assess a solution, reassess the solution, check all s
     const projects: string[] = projectstring.toString().split(",");
     const solutionPage = `=${solutionPath.split("\\").pop()}`;
     console.log(`checking projects for ${solutionNameTagId}`);
-    for (const project of projects) {
+    for (let i = 0; i < 2 && i < projects.length; i++) {
```
Comment (Eruanion) [inline packages/integration-test/src/stability.spec.ts:56]: We only want to iterate over 2 projects maximum here?

**Review by Eruanion — APPROVED** [review]

File: packages/integration-test/src/stability.spec.ts (hunk 2 above)
Comment (huawenm) [reply packages/integration-test/src/stability.spec.ts:57]: yes, running through all projects in nopcommerce may take 3-4 hours,  we currently want to limit the timeout in 2 hours (build exe + run tests).

File: packages/integration-test/src/stability.spec.ts (hunk 1 above)
Comment (cslong) [reply packages/integration-test/src/stability.spec.ts]: add here https://github.com/aws/porting-assistant-dotnet-ui/pull/42/commits/3bb1d45e38a815f8a4d0da47366b3568be1d95dc

**Pull request merged**